Core/include/Acts/Geometry/ProtoLayer.hpp

```diff
@@ -79,24 +79,24 @@ struct ProtoLayer {
   ProtoLayer() = default;
 
   /// Get the parameters : min
-  /// @param bval The accessed binning value
+  /// @param aDir The accessed binning value
```

Comment:
```suggestion
  /// @param aDir The accessed axis value
```

Also in the rest of the file.